package.json

```
@@ -0,0 +1,68 @@
+{
+	"name": "react-payments",
+	"private": true,
+	"version": "0.0.0",
+	"type": "module",
+	"scripts": {
+		"dev": "vite",
+		"build": "tsc && vite build",
+		"lint": "eslint react-payments --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+		"preview": "vite preview",
+		"prettier": "prettier . --write --config .prettierrc",
+		"test": "jest",
+		"test:watch": "jest --watch",
+		"storybook": "storybook dev -p 6006",
```

스토리북을 그냥 실행하면, jest-test-result.json 파일이 없을 수도 있어서 에러가 날 수 있겠네요.
그렇다면 아래와 같이 스토리북 실행 시, 테스트 파일 생성 명령을 매번 먼저 하도록 하는 것도 방법일 것 같아요.
```suggestion
		"storybook": "yarn run test:generate-output && storybook dev -p 6006",
```

---

계속 테스트하며 진행하다보니 생각하지 못한 부분이네요ㅎㅎ

스토리북 실행시 테스트를 먼저 진행하도록 스크립트 수정하였습니다